presto-main/src/main/java/com/facebook/presto/execution/CreateTableTask.java

```diff
@@ -73,12 +81,45 @@ public String explain(CreateTable statement, List<Expression> parameters)
         }
 
         List<ColumnMetadata> columns = new ArrayList<>();
+        Map<String, Object> finalProperties = new HashMap<>();
```

Comment:
This name is bad. It's not really the final properties (not until after the merge call).